src/main/java/com/nike/cerberus/client/auth/aws/StsCerberusCredentialsProvider.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2018 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.client.auth.aws;
+
+import com.amazonaws.DefaultRequest;
+import com.amazonaws.auth.AWS4Signer;
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import com.amazonaws.http.HttpMethodName;
+import com.amazonaws.regions.Regions;
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.nike.cerberus.client.CerberusClientException;
+import com.nike.cerberus.client.auth.TokenCerberusCredentials;
+import com.nike.cerberus.client.http.HttpMethod;
+import com.nike.cerberus.client.http.HttpStatus;
+import com.nike.cerberus.client.model.CerberusAuthResponse;
+import okhttp3.*;
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Provider for allowing users to authenticate with Cerberus with the STS auth endpoint.
+ */
+
+public class StsCerberusCredentialsProvider extends BaseAwsCredentialsProvider {
+
+    protected static String regionName;
```

Comment:
Why is this static?

Reply:
just made it not static, this also seem to fix the failing build